mlir/lib/IR/AsmPrinter.cpp

```diff
@@ -478,6 +478,10 @@ class SymbolAlias {
   bool isType : 1;
   /// A flag indicating whether this alias may be deferred or not.
   bool isDeferrable : 1;
+
```

Comment:
Ok this is basically commit 8773bd0, but without the tests.

Can you elaborate on why you didn't backport the tests?

Reply:
Ah. I think it was just because with all the previous LLVM changes, taking anything but the bare minimum always led to things breaking due to missing stuff introduced in between. For this one I think I didn't even try.